docs/_static/auth/iam.yaml

```diff
@@ -210,52 +210,52 @@ paths:
 #                    type: string
 #        default:
 #          $ref: '../common/error_response.yaml'
-#  /internal/auth/v2/{did}/request-access-token:
-#    post:
-#      operationId: requestAccessToken
-#      summary: Requests an access token using OAuth2.
-#      description: |
-#        Initiates an OAuth2 flow to request an access token from a remote authorization server.
-#
-#        error returns:
-#        * 400 - one of the parameters has the wrong format
-#        * 503 - the authorizer could not be reached or returned an error
-#      tags:
-#        - auth
-#      parameters:
-#        - name: did
-#          in: path
-#          required: true
-#          schema:
-#            type: string
-#            example: did:nuts:123
-#      requestBody:
-#        required: true
-#        content:
-#          application/json:
-#            schema:
-#              required:
-#                - authorizer
-#                - requester
-#                - scope
-#              properties:
-#                authorizer:
-#                  type: string
-#                requester:
-#                  type: string
-#                scope:
-#                  type: string
-#                  description: The scope that will be The service for which this access token can be used. The right oauth endpoint is selected based on the service.
-#                  example: nuts-patient-transfer
-#      responses:
-#        '200':
-#          description: Successful request. Responds with an access token as described in rfc6749 section 5.1.
-#          content:
-#            application/json:
-#              schema:
-#                $ref: '#/components/schemas/TokenResponse'
-#        default:
-#          $ref: '../common/error_response.yaml'
+  /internal/auth/v2/{did}/request-access-token:
```

Comment:
How many APIs do we expect to add on internal/auth? I don't think this specific API conflicts with the auth/v1 api?

Reply:
but two different APIs with the same name also doesn't help....